src/components/LoginWithExternalSiteButton/styles.ts

```diff
@@ -0,0 +1,28 @@
+import { createTheme } from '@mui/material/styles';
+
+import palette from '../../theme/palette';
+
+
+let themeCreate = createTheme({
```

Comment:
define themeCreate as const

Reply:
export const theme = themeCreate;
it needs to be declared before using augmentColor function, so it's necessary to using 'let' keyword